src/dotnet-scaffolding/dotnet-scaffold-aspire/Commands/CachingScaffolderBuilderExtensions.cs

```diff
@@ -0,0 +1,112 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using Microsoft.DotNet.Scaffolding.CodeModification;
+using Microsoft.DotNet.Scaffolding.CodeModification.Helpers;
+using Microsoft.DotNet.Scaffolding.Core.Builder;
+using Microsoft.DotNet.Scaffolding.Core.Steps;
+using Microsoft.DotNet.Tools.Scaffold.Aspire.Helpers;
+using Microsoft.DotNet.Tools.Scaffold.Aspire.ScaffoldSteps;
+
+namespace Microsoft.DotNet.Tools.Scaffold.Aspire.Commands;
+
+internal static class CachingScaffolderBuilderExtensions
```

Comment:
Since we've got a few of these XXXXExtensions classes, I'm thinking they should probably live in an Extensions folder in this project rather than the Commands folder. Though maybe you're planning to remove the Commands folder altogether since you've removed the commands concept now anyway?

Reply:
yep forgot to do that here. Will change it to Extensions